carla/recourse_methods/api/recourse_method.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC, abstractmethod
+
+
+class Recourse_Method(ABC):
```

Comment:
The python standard is to use CapCasing. The google style guide has quite a nice overview: https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations

Reply:
Good to know, I'll change this :+1: